set_gopath.ps1

```diff
@@ -0,0 +1,7 @@
+$goPath = "${pwd}\.gopath"
+$vendorPath = "${pwd}\vendor"
+
+Invoke-Expression "cmd /c rd /s /q $goPath"
+New-Item $goPath\src\github.com\mongodb -ItemType Container
```

Comment:
Use the following to suppress the output.

`[Void](New-Item $goPath\src\github.com\mongodb -ItemType Container)`